presto/dataset.py

```diff
@@ -275,6 +326,7 @@ def __init__(
             dataframe = dataframe[(~dataframe.end_date.dt.year.isin(years_to_remove))]
         self.target_function = target_function if target_function is not None else self.target_crop
         self._class_weights: Optional[np.ndarray] = None
+        self.augment = augment
```

Comment:
should we add logging somewhere that shows that augmentation is enabled when initializing a dataset like that?

Reply:
good point. addressed that here ae27e25db6a172bfac1c1fe0fa3d49861d3780d4